packages/react-client/src/ReactFlightClient.js

```diff
@@ -41,7 +41,7 @@ export type JSONValue =
 
 const PENDING = 'pending';
 const BLOCKED = 'blocked';
-const RESOLVED_MODEL = 'resolved_model';
+const RESOLVED_VALUE = 'resolved_value';
```

Comment:
These are not like the others. E.g. A Module is also a ReactClientValue so why isn't resolved_module also this? We need some way to differentiate between different kinds of values in this context.

Reply:
This comment I found difficult to tackle, and I had a hard time finding the best name for this type of chunk. TL;DR: I went with `RESOLVED_JSON_VALUE`, but I'm am open for other suggestions.

The main problem I have with it is that already before `model` and `module` weren't really the best names. Because as you said, a module is also a `ReactClientValue`, so previously a `Model`. But `ReactClientValue` is named from the perspective for the encoder (flight server or flight reply client), whereas this code here is from the perspective of the decoder (flight client or flight reply server), where `value` (before: `model`) is something else than `ReactClientValue`. The `value` of a module chunk is a client reference object. The `value` of the chunk in question is an `UninitialisedValue`, so in the streaming client it's a JSON `string`, and in the relay clients it's a `JSONValue`. Therefore I went with `RESOLVED_JSON_VALUE`, so a resolved JSON value that can be initialised (i.e. parsed) next. This is also somewhat in line with how it's modeled in the flight server, where there are `completedImportChunks`, `completedJSONChunks` (and `completedErrorChunks`). Maybe the import chunks in the flight server should even be renamed to module chunks?

I'm really glad for the issue you've pointed out here because it made dive deeper into the whole flight implementation, and I now understand a lot more about how RSC works than before.